### DISCUSSION TITLE
Mixed (group) in sex field

### DISCUSSION THREAD

**Discussion by Kema-Ranatunga** [ternaustralia/ontology_tern#203] [Q&A]
Hi TERN folks, 

The field ‘sex’ has four controlled vocabularies: female, hermaphrodite, male & undetermined (in the occurrence_data_1.0.0 template), and we (in ACT) have another vocab listed as ‘mixed (group)’ under sex. This covers both males and females in a group. 
I wonder you are going to consider the mixed group in the controlled vocab in TERN—thanks.

**Comment by edmondchuc** [comment]
Hi @Kema-Ranatunga, we are currently in the process of migrating our TERN controlled vocabularies to GitHub where we can have new proposals such as the `mixed (group)` integrated via pull requests from the community.

I do feel that I am missing a bit of context though. I am not familiar with the `occurrence_data_1.0.0 template`. Is this something you are using to submit data to somewhere? Is it a BDR thing?

**Reply by Kema-Ranatunga** [reply]
Thanks @edmondchuc for your prompt response, and good to hear that you are considering mixed group. 

You are right, the occurrence_data_1.0.0 template is a predefined template created to submit biodiversity data for the Commonwealth BDR

**Reply by edmondchuc** [reply]
I'm trying to work out if we at TERN need to create `mixed (group)` because from your original post, it sounds like `female`, `hermaphrodite`, `male` and `undetermined` are already defined (in the template) but these are not defined by TERN's controlled vocabularies. 

Do you mind uploading the template here? And if possible, can you provide any other relevant documentation or web page links to the submission process?

**Reply by Kema-Ranatunga** [reply]
The following url provides the submission process including the csv template and instructions  

https://staging-bdr.gaiaresources.com.au/#/template/download

**Reply by edmondchuc** [reply]
Thanks for that. These categorical sex values I don't think are defined by TERN. The closest thing I can think of is from a Department project that TERN is working on here that has mixed sexes as a controlled vocabulary. See [Animal sex codes](https://linkeddata.tern.org.au/viewers/dawe-vocabs?uri=https://linked.data.gov.au/def/test/dawe-cv/fcc3a1e1-3e35-4a4f-bd44-eface035025c).

I would suggest to contact Gaia Resources and ask for assistance in what to do if you wanted to submit data with the mixed (group) option.

**Reply by Kema-Ranatunga** [reply]
Many thanks for your comments and help. Pleased to see that the TERN has included mixed sexes. I have already contacted Gaia and will go from there.

**Reply by GrantATgaia** [reply]
Hi @Kema-Ranatunga, my apologies that I've not been back to you sooner. The staging-bdr.gaiaresources.com.au address is intended as an internal (to Gaia) staging environment and Gaia is not currently contracted to support this. I forwarded your support requests onto the department and I gathered Kheeran had been in touch? We're looking forward so much to working with you and others to support and build this capability!

**Reply by Kema-Ranatunga** [reply]
Hi @GrantATgaia, that's alright, and Kheeran and I had a long conversation around these issues. We're looking forward to working closely with you all, thanks again.
